code/game/objects/items/cigs_lighters.dm

```diff
@@ -112,9 +112,11 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	var/icon_off = "cigoff"
 	var/type_butt = /obj/item/cigbutt
 	var/lastHolder = null
-	var/smoketime = 300
+	var/smoketime = 150
 	var/chem_volume = 30
 	var/list/list_reagents = list("nicotine" = 15)
+	var/dragtime = 100
+	var/nextdragtime = 0
 	heat = 1000

```

Comment:
move this above all the `var/`'s

Reply:
I think I did, but honestly I'm not quite sure if that's what you mean since I can't see how the order matters. either way, it's the highest var on the list.

Reply:
Er, meant move the heat var up, the rest were fine wherever really. 

Doesn't matter in terms of performance or anything, just coding standard to have new vars on an object go below vars inherited by it's parent (ie heat would be `/obj/item` level var, so it should be above vars defined by `/obj/item/clothing/mask/cigarette`.)

Reply:
ah yeah I'll do that now.

Reply:
think it's donezo